### ISSUE TITLE
Allow a generation step with multiple tokens

### ISSUE THREAD

**Issue by siddharth9820** [axonn-ai/yalis#22]
We need to support passing multiple tokens in generate for a given sequence. This is needed for spec dec.

**Comment by siddharth9820** [comment]
I am assuming this is resolved now?

**Comment by prajwal1210** [comment]
Yeah I have an implementation for this in spec dec. Not needed as a general feature I think

**Comment by prajwal1210** [comment]
This is now merged as a part of SpecDec
